lib/spack/spack/cmd/mirror.py

```diff
@@ -40,6 +40,16 @@ def setup_parser(subparser):
     create_parser.add_argument(
         'specs', nargs=argparse.REMAINDER,
         help="specs of packages to put in mirror")
+    create_parser.add_argument(
```

Comment:
Yes: I ended up checking that manually vs. using the `add_mutually_exclusive_group` function in `argparse` because it doesn't allow adding non-options (i.e. the specs) to the mutually exclusive group.